.github/workflows/integration_tests.yml

```diff
@@ -203,7 +221,7 @@ jobs:
           --run_id ${{github.run_id}}
 
   build_desktop:
-    name: build-desktop-${{ matrix.os }}-${{ matrix.ssl_variant }}
+    name: build-desktop-${{ matrix.os }}-${{ matrix.ssl_variant }}-${{ matrix.arch }}
```

Comment:
How about reverse the order of the last 2 variables?
build-desktop-${{ matrix.os }}-${{ matrix.arch }}-${{ matrix.ssl_variant }}

Reply:
Done.